chemprop/nn/transforms.py

```diff
@@ -9,9 +9,13 @@
 
 
 class _ScaleTransformMixin(nn.Module):
-    def __init__(self, mean: ArrayLike, scale: ArrayLike):
+    def __init__(self, mean: ArrayLike, scale: ArrayLike, pad: int = 0):
         super().__init__()
 
+        if pad > 0:
+            mean = torch.cat([torch.zeros(pad), torch.tensor(mean, dtype=torch.float)])
+            scale = torch.cat([torch.ones(pad), torch.tensor(scale, dtype=torch.float)])
+
         self.register_buffer("mean", torch.tensor(mean, dtype=torch.float))
         self.register_buffer("scale", torch.tensor(scale, dtype=torch.float))
```

Comment:
IMO it's best to make your broadcasting dimension explicit:
```suggestion
        self.register_buffer("mean", torch.tensor(mean, dtype=torch.float).unsqueeze(0))
        self.register_buffer("scale", torch.tensor(scale, dtype=torch.float).unsqueeze(0))
```